battleship-client/src/main/java/com/academy/solid/nie/client/ui/Cell.java

```diff
@@ -39,24 +39,22 @@ boolean wasShot() {
      * It changes the cell properties.
      *
      * @param newShip
-     * @return
      */
-    Cell addShip(final Ship newShip) {
+    void addShip(Ship newShip) {
         this.ship = newShip;
-        return this;
     }
 
     /**
      * Changes the cell color and marks it as a hit.
      * If there was a piece of a ship in the cell, it would return true.
      *
-     * @return
+     * @return Information whether there was a part of a ship in the cell
```

Comment:
Better wording for all boolean returns is when it returns true (or false).

Reply:
ok